src/main/scala/amailp/intellij/robot/psi/reference/MethodFinderWithoutUnderscoresAndSpaces.scala

```diff
@@ -0,0 +1,43 @@
+package amailp.intellij.robot.psi.reference
+
+import com.intellij.util.Processor
+import com.jetbrains.python.psi.PyElement
+
+/**
+ * @author Sergey Khomutinin skhomuti@gmail.com 
+ *         Date: 30.08.2019
+ */
+class MethodFinderWithoutUnderscoresAndSpaces(reference: String) extends Processor[PyElement] {
+
+  import MethodFinderWithoutUnderscoresAndSpaces._
+
+  var result: PyElement = _
```

Comment:
Should this result be a collection itself? There could be multiple PyElements that match the results (add a test for this also?).

Reply:
Yes, may be. In the current version, the first suitable keyword is taken. However, in such a situation there will be an exception in the robot framework due to the same keyword name in different libraries, and this should be correctly shown by highlighting.
However, it is difficult to implement immediately, because in a robot framework, this conflict can be resolved using the library prefix (`SomeLibrary.Keyword` instead of `Keyword`), which should also be provided for in the plugin. I would like to do this later.

Although there may be an option to leave the choice of library to the user? When clicking on a keyword, show several libraries with a keyword - is this possible in the IDEA? If yes, I think I need help with this.

P.S. I write almost the entire text with the help of a translator, so I apologize for the possible illiteracy :(

Reply:
Ok. I was wrongly thinking of a code completion use case where you have just partial information about the keyword and you need to find all the matching ones.